x/ecocredit/server/core/create_class.go

```diff
@@ -32,8 +32,20 @@ func (k Keeper) CreateClass(goCtx context.Context, req *core.MsgCreateClass) (*c
 		return nil, sdkerrors.ErrUnauthorized.Wrapf("%s is not allowed to create credit classes", adminAddress.String())
 	}
 
+	feeAmt := params.CreditClassFee.AmountOf(req.Fee.Denom)
+	if feeAmt.IsZero() {
+		return nil, sdkerrors.ErrInvalidRequest.Wrapf("%s is not allowed to be used in credit class fees", req.Fee.Denom)
+	}
+	reqFeeAmt, ok := sdk.NewIntFromString(req.Fee.Amount)
+	if !ok {
+		return nil, sdkerrors.ErrInvalidRequest.Wrapf("could not convert %s to %T", req.Fee.Amount, sdk.Int{})
+	}
+	if reqFeeAmt.LT(feeAmt) {
+		return nil, sdkerrors.ErrInsufficientFee.Wrapf("expected %v for fee, got %v", feeAmt, reqFeeAmt)
```

Comment:
Might be nice to include the denom here. The error is:
```
expected 20 for fee, got 10
```

Reply:
https://github.com/regen-network/regen-ledger/pull/912/commits/67d36d01d196abb1c9d66d5b8b5a264a277bc41a#diff-d5c0619debc0b72ca4732464b5573adcf46173b2de79598e66d43da5a2c47202R44

added 👍🏻